apps/docs/src/app/core/component-docs/bar/examples/bar-default-example.component.html

```diff
@@ -15,7 +15,9 @@
             Right Section
         </fd-bar-element>
         <fd-bar-element>
-            <fd-avatar size="xs" image="http://picsum.photos/id/1018/400"> </fd-avatar>
+            <div tabindex="0" role="img" aria-label="Nature">
+                <fd-avatar size="xs" image="http://picsum.photos/id/1018/400"> </fd-avatar>
+            </div>
```

Comment:
Can't you add the tabindex and label on Avatar level? Why do you add additional markup?

Reply:
added tabindex to avatar component itself.